### PR TITLE
[#581] Bottom sheet states Voice Over vocalization

### PR DESCRIPTION
**Pull request by pylapp**
# Issue

See #581

# Definition of Done

- [x] Check any bottom sheets use in components and modules screens (demo app)

# For tests

Using VoiceOver, navigate to the bottom sheet, open and close it, tap and double tap.
There must have several hints and labels (for both opened and closed states and actions)

### CODE REVIEW

File: OrangeDesignSystem/Sources/OrangeDesignSystem/Components/BottomSheet/Internal/ODSBottomSheetStandardModifier.swift [hunk 1]
```diff
@@ -45,45 +56,80 @@ struct ODSBottomSheetStandardHeaderConfig {
...
 
     // ==================
     // MARK: Initializers
     // ==================
 
-    /// Initilize the header of the __ODSBottomSheetStandard__ with title only.
-    /// - Parameters:
-    ///     - title: The title of the bottom sheet
+    /// Initializes the header of the __ODSBottomSheetStandard__ with title only.
+    /// - Parameter title: The title of the bottom sheet
```
Comment (ludovic35) [inline OrangeDesignSystem/Sources/OrangeDesignSystem/Components/BottomSheet/Internal/ODSBottomSheetStandardModifier.swift:69]: missing parameters in documentation

File: OrangeDesignSystem/Sources/OrangeDesignSystem/Components/BottomSheet/Internal/ODSBottomSheetStandardModifier.swift [hunk 2]
```diff
@@ -45,45 +56,80 @@ struct ODSBottomSheetStandardHeaderConfig {
...
+                  icon: icon,
+                  animateIcon: animateIcon)
     }
 
-    private init(title: String, subtitle: String?, icon: Image?, animateIcon: Bool) {
+    /// Initializes the header of the __ODSBottomSheetStandard__ with title, subtitle, icon and accessibility values.
+    ///  - Parameters:
+    ///     - title: The title of the bottom sheet
+    ///     - subtitle: The additional subtitle, optional, default set to nil
+    ///     - stateLabels:The accessibility labels to use for opened and closed states
```
Comment (ludovic35) [inline OrangeDesignSystem/Sources/OrangeDesignSystem/Components/BottomSheet/Internal/ODSBottomSheetStandardModifier.swift:110]: missing space after :

File: OrangeDesignSystem/Sources/OrangeDesignSystem/Components/BottomSheet/Internal/ODSBottomSheetStandardModifier.swift [hunk 3]
```diff
@@ -182,6 +229,83 @@ struct ODSBottomSheetStandard<Content: View>: View where Content: View {
...
+
+    func body(content: Content) -> some View {
+        content
+            .modifier(AccessibilityLabelsModifier(labels: labels, isOpen: isOpen))
+            .modifier(AccessibilityHintsModifier(hints: hints, isOpen: isOpen))
+    }
+
+    /// Just to let developers have in their console some warnings about missing accessibility values.
+    /// Better to have this info in console instead of loosing time in testing.
+    private func checkValues() {
```
Comment (ludovic35) [inline OrangeDesignSystem/Sources/OrangeDesignSystem/Components/BottomSheet/Internal/ODSBottomSheetStandardModifier.swift:257]: rename to logValues and and #if DEBUG

**Review by ludovic35 — CHANGES_REQUESTED** [review]

File: OrangeDesignSystem/Sources/OrangeDesignSystem/Components/BottomSheet/Internal/ODSBottomSheetStandardModifier.swift (hunk 1 above)
Comment (pylapp) [reply OrangeDesignSystem/Sources/OrangeDesignSystem/Components/BottomSheet/Internal/ODSBottomSheetStandardModifier.swift:69]: Où ça ?

Comment (ludovic35) [reply OrangeDesignSystem/Sources/OrangeDesignSystem/Components/BottomSheet/Internal/ODSBottomSheetStandardModifier.swift:69]: Mauvaise lecture, dsl

**Pull request merged**